controller/commentsController.js

```diff
@@ -47,3 +48,42 @@ exports.flagComment = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.content;
+  const owner = req.body.ownerId;
+
+  Comments.findById(comment_id)
+    .exec()
+    .then((comment) => {
+      if (!comment) {
+        return new CustomError(404, "Comment not found");
```

Comment:
You need to call next and pass the CustomError
```suggestion
        return next(new CustomError(404, "Comment not found"));
```